bedrock/base/templates/includes/global-nav.html

```diff
@@ -25,16 +25,16 @@ <h2 class="nav-logo">
           <a href="{{ firefox_url }}" data-id="firefox" data-link-type="nav" data-link-position="top" data-link-name="expand" data-link-group="Firefox">{{ _('Firefox') }}</a>
         </li>
         <li class="item-internet-health">
-          <a href="{{ url('mozorg.internet-health') }}" data-id="internet-health" data-link-type="nav" data-link-position="top" data-link-name="expand" data-link-group="Internet Health">{{ _('Internet Health') }}</a>
+          <a href="{{ url('mozorg.internet-health') }}" data-link-type="nav" data-link-position="top" data-link-name="Internet Health" data-link-group="Internet Health">{{ _('Internet Health') }}</a>
```

Comment:
Do you think we still need `data-link-group`? It's now a repeat of `data-link-name`...